forms-flow-api/src/formsflow_api/resources/employeeData.py

```diff
@@ -37,3 +37,24 @@ def get():
             return err.error, err.status_code
         
         return userData, HTTPStatus.OK
+
+
+@cors_preflight("GET, OPTIONS")
+@API.route("/names", methods=["GET", "OPTIONS"])
+class EmployeeNames(Resource):
+    """Resource for managing employee names"""
+
+    @staticmethod
+    @profiletime
+    @auth.require
+    def get():
+        """Get employee names from ODS."""
+        args = request.args
+        print(args)
```

Comment:
Is this `print` intended?

Reply:
Good catch, No it is not. Just Removed the print from both files.